launcher-scripts/Windows-launcher.bat

```diff
@@ -1,7 +1,9 @@
 @echo off
-set OLD_JAVA_HOME=%JAVA_HOME%
-set BAT_DIRECTORY=%~dp0
-set JAVA_HOME=%BAT_DIRECTORY%jdk8u242-b08-jre
+setlocal EnableDelayedExpansion
```

Comment:
I wonder if we need this. Only because I've not seen it needed in other wrappers before, and previously below there was only quotes around the `start` statement so maybe the issue was simply the values for the variables need quoting?

Reply:
This was me attempting to get it to work for ALL special characters, however not even this will do that. 
`adminconsole ^=&[]{}()-_ +,~ !'+,`~%@#$Ó` is in my path and works fine with or without delayed expansion, but if there is ever a semicolon it will not open no matter what I do, delayed expansion or no. 
I'm thinking, we could just add to the readme a note to make sure no semicolons are in the folder path?
Either way, EnableDelayedExpansion doesn't seem to fix the semicolon issue, so I'll remove it.

Reply:
I think that's a fine limitation, and indeed wouldn't even need documenting.

Reply:
Okay cool, thanks!